addons/website/views/website_templates.xml

```diff
@@ -315,6 +315,14 @@
     </xpath>
 </template>
 
+<template id="header_button" inherit_id="website.layout" name="Header Button" active="False">
+    <xpath expr="//ul[@id='top_menu']" position="inside">
+        <li class="nav-item align-self-md-center ml-1 o_header_menu_button">
```

Comment:
Using this class is what I was talking about. You did not need 10 lines of css for this, just use the right class :wink:

Reply:
you suggested "align-self-center" but I didn't know we could add "md-" ;)